contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -402,16 +438,52 @@ contract StakeManager is
         bool acceptDelegation,
         bytes calldata signerPubkey
     ) external {
+        _dethroneAndStake(auctionUser, heimdallFee, validatorId, auctionAmount, acceptDelegation, signerPubkey, false);
+    }
+
+    // @note
+    function dethroneAndStakeLegacy(
```

Comment:
can be removed since auctions are disabled

Reply:
True, I will do that.